src/patternfly/components/Badge/badge.hbs

```diff
@@ -1,5 +1,10 @@
 <span class="{{pfv}}badge
-  {{#if badge--IsUnread}}pf-m-unread {{/if}}
+  {{#if badge--IsUnread}}
+    pf-m-unread 
+  {{else}}
+    pf-m-read 
+  {{/if}}
```

Comment:
This assumes `badge--IsUnread` is used to set read/unread, so it's going to add `.pf-m-read` if a badge used either `badge--modifier="pf-m-read"` or `badge--modifier="pf-m-unread"`. The badge examples in this PR look good since they have all been updated to use `badge--IsUnread` but there are a lot of other examples in the workspace that will need to be updated, too. These examples probably look fine, but the classlist has both `.pf-m-read` (from `! badge--IsUnread`) as well as whatever is in `badge--modifier`.  I just searched for `badge--modifier` to get this list. Totally fine if you'd prefer do this as a follow-up, too, so we can merge this PR. 

```
components/Breadcrumb/examples/Breadcrumb.md:            {{#> badge badge--modifier="pf-m-unread"}}
components/Button/examples/Button.md:    {{#> badge badge--modifier="pf-m-read"}}
components/Button/examples/Button.md:    {{#> badge badge--modifier="pf-m-read"}}
components/Button/examples/Button.md:    {{#> badge badge--modifier="pf-m-unread"}}
components/Button/examples/Button.md:    {{#> badge badge--modifier="pf-m-unread"}}
components/DualListSelector/dual-list-selector-item.hbs:        {{#> badge badge--modifier="pf-m-read"}}
components/Menu/templates/menu-breadcrumbs--dropdown.hbs:        {{#> badge badge--modifier="pf-m-read"}}
components/MenuToggle/examples/MenuToggle.md:    {{#> badge badge--modifier="pf-m-unread"}}
components/MenuToggle/examples/MenuToggle.md:    {{#> badge badge--modifier="pf-m-unread"}}
components/MenuToggle/menu-toggle.hbs:      {{> badge badge--modifier="pf-m-unread" badge--text=menu-toggle--badge--count}}
components/NotificationDrawer/examples/NotificationDrawer.md:              {{#> badge badge--modifier="pf-m-unread"}}2{{/badge}}
components/NotificationDrawer/examples/NotificationDrawer.md:              {{#> badge badge--modifier="pf-m-unread"}}2{{/badge}}
components/NotificationDrawer/examples/NotificationDrawer.md:              {{#> badge badge--modifier="pf-m-unread"}}2{{/badge}}
components/NotificationDrawer/examples/NotificationDrawer.md:              {{#> badge badge--modifier="pf-m-unread"}}3{{/badge}}
components/TextInputGroup/text-input-group--search-input.hbs:        {{> badge badge--modifier="pf-m-read" badge--text=text-input-group--search-input--count}}
components/TreeView/examples/TreeView.md:                        {{#> badge badge--modifier="pf-m-read"}}
components/TreeView/examples/TreeView.md:                    {{#> badge badge--modifier="pf-m-read"}}
components/TreeView/examples/TreeView.md:                    {{#> badge badge--modifier="pf-m-read"}}
components/TreeView/examples/TreeView.md:                {{#> badge badge--modifier="pf-m-read"}}
components/TreeView/examples/TreeView.md:                {{#> badge badge--modifier="pf-m-read"}}
components/TreeView/examples/TreeView.md:            {{#> badge badge--modifier="pf-m-read"}}
components/TreeView/examples/TreeView.md:            {{#> badge badge--modifier="pf-m-read"}}
components/TreeView/examples/TreeView.md:            {{#> badge badge--modifier="pf-m-read"}}
components/TreeView/examples/TreeView.md:            {{#> badge badge--modifier="pf-m-read"}}
demos/NotificationDrawer/notification-drawer-demo-groups.hbs:              {{#> badge badge--modifier="pf-m-unread"}}2{{/badge}}
demos/NotificationDrawer/notification-drawer-demo-groups.hbs:              {{#> badge badge--modifier="pf-m-unread"}}2{{/badge}}
demos/NotificationDrawer/notification-drawer-demo-groups.hbs:              {{#> badge badge--modifier="pf-m-unread"}}2{{/badge}}
demos/NotificationDrawer/notification-drawer-demo-groups.hbs:              {{#> badge badge--modifier="pf-m-unread"}}3{{/badge}}
demos/NotificationDrawer/notification-drawer-template-groups.hbs:              {{#> badge badge--modifier="pf-m-unread"}}2{{/badge}}
demos/NotificationDrawer/notification-drawer-template-groups.hbs:              {{#> badge badge--modifier="pf-m-unread"}}2{{/badge}}
demos/NotificationDrawer/notification-drawer-template-groups.hbs:              {{#> badge badge--modifier="pf-m-unread"}}2{{/badge}}
demos/NotificationDrawer/notification-drawer-template-groups.hbs:              {{#> badge badge--modifier="pf-m-unread"}}3{{/badge}}
demos/NotificationDrawer/notification-drawer-template.hbs:  page-template-header-tools-notification-badge--badge--modifier="pf-m-unread"
```

Reply:
Thanks - to avoid these additional implications I've backed out the `if/else` statement.  

Separately I did notice that the screen-reader text was only checking for `if IsUnread` so have added an `unless IsDisabled`